fileio/easyio.go

```diff
@@ -80,8 +80,7 @@ func EasyPeekReal(file *EasyReader, n int) ([]byte, error) {
 
 // EasyRemove deletes the input file.
 func EasyRemove(filename string) {
-	err := os.Remove(filename)
-	exception.PanicOnErr(err)
+	MustRemove(filename)
```

Comment:
Should EasyRemove disappear?  Seems like places that call EasyRemove could be changed to call MustRemove

Reply:
It could, though the same would be true of EasyPeekReal, EasyNextRealLine, and EasyNextLine which just wrap the non-easy version of the function.